src/internal/views/nunjucks/billing/two-part-tariff-review.njk

```diff
@@ -0,0 +1,89 @@
+
+{% block content %}
+<div class="govuk-grid-row govuk-!-margin-bottom-0">
+  <div class="govuk-grid-column-two-thirds">
+    <section class="govuk-!-margin-bottom-6">
+      {% set caption %}
+        {{ 'Bill run ' + batch.externalId if batch.externalId else null }}
+      {% endset %}
+        {{ title(pageTitle, 'This is the caption', null, true, true) }}
+    </section>
+  </div>
+</div>
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
+<div class="govuk-grid-column-two-thirds">
+      <p class="govuk-body">
+        {{ badge(batch | batchBadge(false)) }}
+      </p>
+      <div class="govuk-grid-row">
+        {{ batchMetadata(batch) }}
+      </div>
+  </div>
+</div>
+ 
+<section class="govuk-!-margin-bottom-6">
+  {{
+    govukButton({
+      text: "Cancel bill run",
+      href: '/billing/batch/' + batch.id + '/cancel',
+      classes: "govuk-button--secondary govuk-!-margin-left-1"
+    })
+  }}
+</section>
+
+<section class="govuk-!-margin-bottom-9">
+  <h2 class="govuk-heading-l" id="issuesTitle">{{ totals.errors }} licences have data issues</h2>
+  <div class="table govuk-!-margin-bottom-6" id="dataIssues">
+    {% if licences.length > 0 %}
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Licence</th>
+            <th scope="col" class="govuk-table__header">Licence Holder</th>
+            <th scope="col" class="govuk-table__header">Issue</th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric">Action</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          {% for licence in licences %}
+            {% if licence.twoPartTariffStatuses.length > 0 %}
+              <tr class="govuk-table__row">
+                <td class="govuk-table__cell">
+                  {{ licence.licenceNumber }}
+                </td>
+                <td class="govuk-table__cell">
+                  {{ licence.licenceHolder.salutation }}
+                  {{ licence.licenceHolder.initials }}
+                  {{ licence.licenceHolder.firstName }}
+                  {{ licence.licenceHolder.lastName }}
+                </td>
+                <td class="govuk-table__cell">
+                  {{ licence.twoPartTariffStatuses }}
+                </td>
+                <td class="govuk-table__cell govuk-table__cell--numeric">
+                  <a href="#" class="govuk-link">Review</a>
+                </td>
+              </tr>
+            {% endif %}
+          {% endfor %}
+        </tbody>
+      </table>
+    {% endif %}
+  <div>
+</section>
+<section class="govuk-!-margin-bottom-0v">
+<h2 class="govuk-heading-l" id="readyTitle">
+  {{ licences.length - totals.errors}} licences are ready for billing
+</h2>
+<p class="govuk-body govuk-!-margin-bottom-9">
+<a href="#" class="govuk-link">View licences ready for billing</a>
+</p>
+</section>
```

Comment:
I think there is a bit missing here - see https://wrls-prototype.herokuapp.com/bd/charges-2020/two-part-review

You have resolved all returns data issues. Continue to generate bills.
Continue Cancel bill run

I think probably the 'Cancel bill run' option should always be visible - perhaps we should check with UCD on this point

Reply:
There are 2 cancel bill run buttons on this screen only 1 being visible  at a time. The top one when there are errors to resolve you can cancel the bill run. This top button then disappears if all the errors are resolved and the bottom cancel bill run button appears with the continue button.